web.test.tests/web.test.tests/Tests/HistoryPageTests.cs

```diff
@@ -0,0 +1,134 @@
+        }
+
+        [Test]
+        public void HistoryNotRestored()
+        {
+            //Act
+            DepositPage depositPage = new DepositPage(driver);
+            depositPage.SetAmountRateTerm(12345, 87, 299);
+            depositPage.SelectFinYear(360);
+            depositPage.Calculate();
+
+            //Assert
+            Assert.IsTrue(depositPage.OpenHistoryPage().Clear().BackToDeposit().OpenHistoryPage().IsHistoryClean());
+        }
+
+        [TestCase(5)]
+        public void OrderingInHistory(int amount_of_calculations)
+        {
+            //Act
+            String[] entered_values = new string[8];
+            DepositPage depositPage = new DepositPage(driver);
+            for (int i = 0; i < amount_of_calculations; i++)
+            {
+                depositPage.SetAmountRateTerm(12341 + i, 91 + i, 301 + i);
+                depositPage.Calculate();
+                if (i == 0) entered_values = depositPage.CaptureDataToArray();
+            }
+
+            //Assert
+            Assert.AreEqual(entered_values, depositPage.OpenHistoryPage().CaptureRowToArray(amount_of_calculations));
+        }
+
+        [TestCase(5)]
+        public void SequentialCalculationsSaved(int amount_of_calculations)
+        {
+            //Act
+            DepositPage depositPage = new DepositPage(driver);
+            depositPage.OpenHistoryPage().Clear().BackToDeposit();
+            for (int i = 0; i < amount_of_calculations; i++)
+            {
+                depositPage.SetAmountRateTerm(12341 + i, 91 + i, 301 + i);
+                depositPage.Calculate();
+            }
+            
+            //Assert
+            Assert.AreEqual(amount_of_calculations, depositPage.OpenHistoryPage().GetRowsNumber());
+        }
+
+        [TestCase("dd/MM/yyyy", 11)]
+        [TestCase("dd-MM-yyyy", 12)]
+        [TestCase("MM/dd/yyyy", 13)]
+        [TestCase("MM dd yyyy", 14)]
+        public void DatesUpdatedAccordingToDateFormatSetting(String dateFormat, int term)
+        {
+            //Arrange
+            DepositPage depositPage = new DepositPage(driver);
+            DateTime startDate = depositPage.StartDate;
+            DateTime endDate = startDate.AddDays(term);
+            String strStartDate = startDate.ToString(dateFormat, CultureInfo.InvariantCulture);
+            String strEndDate = endDate.ToString(dateFormat, CultureInfo.InvariantCulture);
+            depositPage.SetAmountRateTerm(1, 1, term);
+            depositPage.Calculate();
+
+            //Act
+            SettingsPage settingsPage = depositPage.OpenSettingsPage();
+            settingsPage.SelectDateFormat.SelectByText(dateFormat);
+            settingsPage.Save();
+            HistoryPage historyPage = depositPage.OpenHistoryPage();
+            String histStartDate = historyPage.GetDateFrom(1);
+            String histEndDate = historyPage.GetDateTo(1);
+            
+            //Assert
+            Assert.AreEqual(strStartDate,histStartDate);
```

Comment:
Please add 'space' for better readability.

```suggestion
            Assert.AreEqual(strStartDate, histStartDate);
```

Reply:
Done